string name
trajectory_msgs/JointTrajectory joint_trajectory
PoseTrajectory pose_trajectory